queries/nim_format_string/highlights.scm

```diff
@@ -0,0 +1,19 @@
+(string_literal)
+(matching_curlies 
+  opening_curly: (opening_curly) @punctuation.special 
+  equals: (equals)? @punctuation.special
+  closing_curly: (closing_curly) @punctuation.special)
+
+(format_specifiers
+  colon: (colon) @punctuation.delimiter
+  fill_align: (fill_align)? @conditional.ternary
+  sign: (sign)? @operator
+  hash: (hash)? @punctuation.special
+  zero: (zero)? @field
+  min_width: (min_width)? @number
+  precision: (precision)? @number
+  type: (type)? @type)
+
+(matching_curlies 
+  nim_expression: (nim_expression 
+    escaped_curly: (escaped_curly)+ @string.escape) @error)
```

Comment:
Why `@error`?

Reply:
Do not use `@error`; that's a purely diagnostic capture that should not be used for actual highlights. Use `@text.danger` for error-type text instead.

Reply:
alright. gonna go back to `@none` then

Reply:
What does that do? Why not simply skip the capture?

Reply:
(but, yes, `@none` is usually used for string interpolation, if that's what's happening here.)

Reply:
https://github.com/aMOPel/tree-sitter-nim-format-string#limitations

It is supposed to highlight the nim expression as white or whatever (instead of string color), when we are not injecting the nim parser.

Reply:
Then use `@none`.